### PR TITLE
Add .travis.yml

### PR DESCRIPTION
**Pull request by beaugunderson**
You can verify that the [build was successful on my fork](https://travis-ci.org/beaugunderson/viper/builds/258661293).

Closes #292 .

### CODE REVIEW

**Comment by beaugunderson** [comment]
Closes #293.

**Comment by yograterol** [comment]
@beaugunderson great! Coud you add the travis integration with the README.md?

File: setup.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
     license=license,
     packages=find_packages(exclude=('tests', 'docs')),
     install_requires=[
-        'ethereum == 1.3.7',
+        'ethereum==2.0.4',
         'serpent',
```
Comment (yograterol) [inline setup.py]: It would be good to add the version for every dependency, what do you think?

Comment (beaugunderson) [inline setup.py]: Yes, I think that's a best practice; though `serpent` here is incorrect (it should be `ethereum-serpent`) and testing requirements should not be in `install_requires`, they should actually be in `tests_require`.

Comment (yograterol) [inline setup.py]: We could to implement [Semantic Version](http://semver.org/) and freeze the dependencies version with every Viper version.

**Comment by beaugunderson** [comment]
@yograterol OK, I added the Travis badge ✨

File: setup.py (hunk 1 above)
Comment (beaugunderson) [inline setup.py]: 👍 for using semver

**Pull request merged**